README.md

```diff
@@ -8,6 +8,7 @@ An open-source documentation about Android UI testing:
 2. Practices: Code style
 3. Practices: Run UI tests on obfuscated build
```

Comment:
what about enumerating this with more indentation? like this

### Roadmap
1. Adoption: Where it all starts
2. Practices: 
   - Code style
   - Run UI tests on obfuscated build 
   - Shared UI tests guide
   - Test reports
   
Moreover, it might make sense to add some info about the creation of repos with samples?

Reply:
Good remarks, will be fixed 👍

Reply:
fixed